### DISCUSSION TITLE
SuperMap 引入是黑屏的

### DISCUSSION THREAD

**Discussion by Jiachang-Cai** [zouyaoji/vue-cesium#372] [Q&A]
SuperMap 引入是黑屏的
<img width="1274" alt="image" src="https://user-images.githubusercontent.com/8606851/226952362-cc463278-11db-4ef4-a694-39eba4e1e6a5.png">

**Comment by zouyaoji** [comment]
请问你打开的哪儿的场景呢？

**Reply by Jiachang-Cai** [reply]
[[你的 示例](https://zouyaoji.top/vue-cesium/#/zh-CN) ](https://zouyaoji.top/vue-cesium/#/zh-CN/component/platforms/vc-demo-supermap) 就demo 就是 我自己的场景也是黑的

**Reply by zouyaoji** [reply]
我看正常的啊，你更新一下浏览器，显卡驱动啥的。
![image](https://user-images.githubusercontent.com/24430116/227410038-38bbd610-1b6d-47ef-8ab5-754e38f63e92.png)
